conda/recipes/pylibwholegraph/meta.yaml

```
@@ -75,4 +75,6 @@ requirements:
 
 about:
   home: https://rapids.ai/
+  license: Apache-2.0
+  license_file: ../../../LICENSE
```

```suggestion
  license_file: LICENSE
```

The project already has a symlink resolving this relative path: https://github.com/rapidsai/wholegraph/blob/branch-24.10/python/pylibwholegraph/LICENSE

I think it'd be simpler to just make this `LICENSE`, as we do in `cudf`:

https://github.com/rapidsai/cudf/blob/ad1369d2d6eabf4b0ae480a10463a74f3034aece/conda/recipes/cudf/meta.yaml#L122